Sharpmake.Generators/VisualStudio/Vcxproj.cs

```diff
@@ -174,7 +174,10 @@ public class CombinedCustomFileBuildStep
             foreach (var customBuildStep in buildSteps)
             {
                 var relativeBuildStep = customBuildStep.MakePathRelative(resolver, (path, commandRelative) => Util.SimplifyPath(Util.PathGetRelative(referencePath, path)));
-                relativeBuildStep.AdditionalInputs.Add(relativeBuildStep.Executable);
+                if(customBuildStep.DependOnExecutable)
+                {
+                    relativeBuildStep.AdditionalInputs.Add(relativeBuildStep.Executable);
```

Comment:
just to confirm. This is problematic because the compiler won't be able to check timestamps on the executable because it won't be able to find it?

Reply:
This actually just shows up as one warning that visual studio couldn't find the file and incremental builds might not work.